README.md

```diff
@@ -1,9 +1,9 @@
 # ESLint RFCs
 
-Many changes, including bug fixes and documentation improvements can be
+Many changes, including bug fixes and documentation, improvements can be
```

Comment:
I think the comma needs to move after "improvements" here.

Reply:
Can you submit "suggested changes" for typos? Saves me from needing to redo the pull request each time.

Reply:
Sure, I haven't tried that feature but I'll see if I can figure it out.